src/tasks/AndroidAppBuilder/MSBuild_Logs/MSBuild_pid-31830_2ec316d3428a488197c2e1d291e38c0f.failure.txt

```diff
@@ -0,0 +1,9 @@
+UNHANDLED EXCEPTIONS FROM PROCESS 31830:
```

Comment:
@agocke extraneous file.

we could put failure.txt in our .gitignore, but there's a bug open to fix this in MSBuild